core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/MetadataSetupService.java

```diff
@@ -317,8 +326,17 @@ public void saveLoadBalancerMetadata(Stack stack, Iterable<CloudLoadBalancerMeta
                 String endpoint = loadBalancerConfigService.generateLoadBalancerEndpoint(stack);
                 LOGGER.info("Saving load balancer endpoint as: {}", endpoint);
                 loadBalancerEntry.setEndpoint(endpoint);
+                loadBalancerEntry.setProviderConfig(loadBalancerConfigConverter.convertLoadBalancer(stack.getCloudPlatform(),
```

Comment:
for a separate PR: could you please move out LB related stuff from `MetadataSetupService` to a separate class? like `LBMetadataSetupService`

Reply:
Created CB-13645.